tests/unit/conftest.py

```diff
@@ -426,4 +430,26 @@ def package_service(default_project, default_factory):
     )
 
 
-# pylint: enable=import-outside-toplevel
```

Comment:
why did you remove this line?

Reply:
Oh, I did not notice it was disabled above. Will add back.